CMakeLists.txt

```diff
@@ -1,52 +1,54 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(laser_line_extraction)
 
-find_package(catkin REQUIRED COMPONENTS
-  cmake_modules
-  geometry_msgs
-  message_generation
-  roscpp
-  rospy
-  sensor_msgs
-  visualization_msgs
-)
+find_package(catkin QUIET COMPONENTS cmake_modules geometry_msgs 
+    message_generation roscpp rospy sensor_msgs visualization_msgs)
 
 find_package(Eigen3 REQUIRED)
 
-add_message_files(
-  FILES
-  LineSegment.msg
-  LineSegmentList.msg
-)
+if (catkin_FOUND)
+    message("-- Catkin found. Compiling ROS components.")
```

Comment:
Can this line be changed to 
```
message(STATUS "-- Catkin found. Compiling ROS components.")
```
Without a severity indication, catkin tools (a ROS tool) considers these messages warnings and you get annoying displays.
PS. Any reason to start with two dashes? Just curious

Reply:
I used the two lines to make it look like STATUS but that is a better way to do it, and it now shows up with the two dashes like the other messages.

Reply:
That's quick!
PS. I am just an interested 3th party with no authority here. So you'll have to wait for someone here for some approval / comments I guess.